.gitattributes

```diff
@@ -1 +1,2 @@
 docker/**/*.sh text eol=lf
+*.svg binary
```

Comment:
Unrelated bycatch, I assume..

Reply:
Yup, Ill search for say a model name and the erd.svg muddies my git grep